features/steps/steps.py

```diff
@@ -137,6 +148,23 @@ class representation_type_error:
     def __str__(self):
         return f"On instance {fmt(self.inst)} the {self.representation_id} shape representation does not have {self.representation_type} as RepresentationType"
 
+@dataclass 
+class value_identical_unique_error:
+    related: ifcopenshell.entity_instance 
```

Comment:
I don't understand here, which is `related` and which is `relating`?

Reply:
The idea was to always go from the top attribute to the bottom in the feature file, so to speak. So, in this case
`IfcGeometricRepresentationContext -> HasCoordinateOperation > IfcMapConversion`
Related in this case is IfcGeometricRepresentationContext (first), and relating IfcMapConversion.
It is consistent with ALB002, for example:
`IfcAlignment (top; related) > IfcAlignmentVertical (bottom; relating)`
 
With this rule, it is perhaps a bit confusing as 'relating' (e.g. #22IfcMapConversion...) is very closely linked to the value (IfcMapConversion). See also comment below

Reply:
I realize that it is consistent with ALB002 indeed, but consistently wrong .. 

> In case of the 1-to-many relationship, the related side of the relationship shall be an aggregate being a set 1:N

It was good initially, but it got reversed with the Nests - IsNestedBy confusion. Will change it here and make a jira ticket to set it correct for the whole file.

Reply:
Reversed in https://github.com/buildingSMART/ifc-gherkin-rules/pull/35 and added a note at other lines were it needs to be reversed.

Reply:
The variable names are changed to more sensible names, I think. It should be clearer now. https://github.com/buildingSMART/ifc-gherkin-rules/pull/35/commits/2997a98dc1769acf00a07a37cf62d971b12aa7dd